### ISSUE TITLE
Sync blocking lists between instances

### ISSUE THREAD

**Issue by stemy2** [tootsuite/mastodon#12956]
### Pitch

I'd like to sync my instance blocking list with another instance i trust.

### Motivation

This is fastidious to check manually blocking lists from other instances and then copy-paste in mine.

**Comment by busshimoe** [comment]
duplicate of #11510

**Comment by Shleeble** [comment]
yeah. I'd love to see block of instances and suspends of users broadcast to other servers.
